packages/server/tests/SocketonDisconnection.test.ts

```diff
@@ -135,23 +154,25 @@ test.group('Rooms life cycle', (group) => {
             initialTracksIDs: [],
         });
         socket.emit('MTV_CREATE_ROOM', settings);
-        await sleep();
-        await sleep();
-        const roomBefore = await MtvRoom.findBy('creator', userID);
-        assert.isNotNull(roomBefore);
-        //As sinon mocks the whole thing synchrounously we cannot trust the order
-        assert.isTrue(
-            receivedEvents.includes('MTV_CREATE_ROOM_SYNCHED_CALLBACK'),
-        );
-        assert.isTrue(receivedEvents.includes('MTV_CREATE_ROOM_CALLBACK'));
+
+        await waitFor(async () => {
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** 👍